nox/main.py

```diff
@@ -263,8 +265,17 @@ def main():
     parser.add_argument(
         'posargs', nargs=argparse.REMAINDER,
         help='Arguments that are passed through to the sessions.')
+    parser.add_argument(
+        '--version', action='store_true',
+        help='Output the nox version and exit.')
 
     args = parser.parse_args()
+
+    if args.version:
+        dist = pkg_resources.get_distribution('nox-automation')
+        sys.stderr.write("{}\n".format(dist.version))
```

Comment:
Why write to stderr? Also, you could instead be a bit more "modern" and idiomatic:

```
from __future__ import print_function

print('blah', file=sys.stderr)
```